data/crac/crac-api/src/main/java/com/powsybl/openrao/data/cracapi/networkaction/DanglingLineActionAdder.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openrao.data.cracapi.networkaction;
```

Comment:
Here and for all adders, update copyright and author.

Reply:
Done for all new files